infrastructure/comms/src/connection/peer_connection/connection.rs

```diff
@@ -53,12 +64,31 @@ impl fmt::Display for PeerConnectionState {
         use PeerConnectionState::*;
```

Comment:
A readability nit: I think it's more readable if you explicitly use PeerConnectionState::Connecting and 'PeerConnectionState::Connected below rather than using this `use ...` statement. It's not clear where these values come from or that they are even from by using the * include.